Project.toml

```diff
@@ -9,7 +9,9 @@ CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 ClimaComms = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
 ClimaCoreTempestRemap = "d934ef94-cdd4-4710-83d6-720549644b70"
+ClimaTimeSteppers = "595c0a79-7f3d-439a-bc5a-b232dc3bde79"
```

Comment:
a mental note that once we move the RK4 tableau to CTS, I think we can remove this dependency.

for now, we just need to add compat entries to keep the Julia registry happy

Reply:
CTS dependency removal noted in #219